### PR TITLE
[DASH-2240] Add possibility to open hosting in new tab from files view

### PR DESCRIPTION
**Pull request by gh3r**
screen shot 2016-10-06 at 14 48 33 2

### CODE REVIEW

File: src/apps/Hosting/Files.js [hunk 1]
```diff
@@ -89,12 +120,12 @@ const HostingFilesView = React.createClass({
         <InnerToolbar title="Website Hosting">
           <Show if={items.length && !isLoading}>
             <RaisedButton
-              label={isDefaultHosting ? 'Published' : 'Publish'}
-              onTouchTap={this.handleShowPublishDialog}
+              label="Open in tab"
```
Comment (wpasinski) [inline src/apps/Hosting/Files.js]: but what do we open with that button? shouldn't it say something like _Go to site_ or something like this?

**Comment by gh3r** [comment]
@wpasinski Review applied changes

File: src/apps/Hosting/Files.js [hunk 2]
```diff
@@ -89,12 +118,12 @@ const HostingFilesView = React.createClass({
...
             <RaisedButton
-              label={isDefaultHosting ? 'Published' : 'Publish'}
-              onTouchTap={this.handleShowPublishDialog}
+              label="Go to site"
               primary={true}
-              disabled={isDefaultHosting}
-              disabledBackgroundColor={Colors.green500}
-              disabledLabelColor="#FFF"
+              icon={<FontIcon className="synicon-open-in-new" style={{ marginTop: 4 }} />}
+              onTouchTap={!hasHostingUrl && this.showSnackbar}
```
Comment (gh3r) [inline src/apps/Hosting/Files.js]: Created handler as suggested. And renamed `showSnackbar` method. When i moved `setSnackbarNotification` inside of handler it was triggered when user entered page. So let's leave it as it is.

**Comment by gh3r** [comment]
@wpasinski @patrykkopycinski Please review once more

**Pull request merged**